wca-regulations.md

```diff
@@ -248,7 +248,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 9f14) For "Mean of 3" and "Average of 5" rounds, if two or more competitors achieve identical average/mean results, rankings are assessed based on the best attempt per competitor, where "better" is defined as the shorter time.
     - 9f15) Competitors who achieve the same result in a round receive an identical ranking for the round.
 - 9g) A Combined Round consists of two phases of attempts, where competitors advance to the second phase if they meet a designated cutoff during the first phase.
-    - 9g2) Whether a competitor proceeds to next phase of a Combined round, must be decided by ranking (best x competitors) or by result (all competitors with a best result under x) of the first phase.
+    - 9g2) Whether a competitor proceeds to next phase of a Combined round, must be decided by ranking (best x competitors) or by result (all competitors with a better result than x) of the first phase.
```

Comment:
Might it make sense to link to [9f12](https://www.worldcubeassociation.org/regulations/#9f12) here after the use of the word "better"?

Reply:
@thewca/wrc-team Thoughts? I personally think it's a good idea!

Reply:
Bump, this looks like it got merged without this chance happening. Thoughts?

Reply:
Yeah, I completely missed that before merging. Thanks for checking, @jfly

Reply:
Addressed on #478